docs/06-contributors/999-rfcs/000-stories/story-13.md

```diff
@@ -0,0 +1,169 @@
+# User Story 13 - Task List with uuid, Json and duration
+
+> **Status**: Expected released on 2023/02/16
+
+The following code is an iteration of Wing's task list.
+This version of it includes the following functional changes:
+- Using npm's uuid package inflight instead of the counter resource 
+- Storing Json instead of a string
+- Using duration for estimated effort
+
+Also, it includes some non-functional requirements: 
+- The Bucket should be replicated across different regions 
```

Comment:
Should the story indicate that this is toggled featured rather than change in implementation of underlying resources?

Reply:
I think this should be a plugin and the story should be explicit about how this plugin is applied.